test/test_options.py

```diff
@@ -355,7 +355,7 @@ def test_versioned_component_long_help():
         "  -e, --extra-args EXTRA_ARGS     Extra arguments to pass to the install\n"
         "                                  command\n"
         "  -m, --method [auto|apt|brew|neurodebian|deb-url|autobuild|snapshot|"
-        "conda|datalad/git-annex|datalad/packages|dmg]\n"
+        "conda|datalad/git-annex|datalad/git-annex:latest|datalad/packages|dmg]\n"
```

Comment:
Not to bring pain, but I was thinking -- should we may be actually make prior one into `:stable` (or :tested or smth like that) and have this "new" one as datalad/git-annex?
  
```suggestion
        "conda|datalad/git-annex:stable|datalad/git-annex|datalad/packages|dmg]\n"
```

it might actually be better aligned with others, where there might have not been any testing against datalad etc.  WDYT @jwodder?

Reply:
Either way is fine with me.

Reply:
Let's then go for 
```suggestion
        "conda|datalad/git-annex:tested|datalad/git-annex|datalad/packages|dmg]\n"
```
since we can't make claims for stability etc.

Reply:
Done.

Reply:
thank you! Let's then proceed with the release and see where it would bring us in datalad-extensions: today's round (after switching to use datalad-installer with datalad/git-annex) went through well, besides @mih 's extras: https://github.com/datalad/datalad-extensions/#datalad-extensions-status  so any new fails would be likely due to anything "new" in git-annex.